### ISSUE TITLE
Bundle example 2 is not correct

### ISSUE THREAD

**Issue by yunwwang** [smart-on-fhir/health-cards#118]
https://smarthealth.cards/examples/example-02-a-fhirBundle.json

- It does not have Patient entry
- Almost all the entries having reference to `resource:0` which is DiagnosticReport is the bundle. Even the DiagnosticReport.subject reference to itself.
-

**Comment by jmandel** [comment]
Yeah, we probably want to hand-craft a longer example; this one only exists to demonstrate the QR-splitting mechanism. Would you be up for creating a better demo bundle for this? Maybe starting from the HL7 example but crafting something we can include directly in this repo @yunwwang?
